### PR TITLE
fix(firestore-bigquery-export): pass in final path of collection path to collectionGroup

### PR DESCRIPTION
**Pull request by cabljac**
This implements the fix suggested in https://github.com/firebase/extensions/issues/1247 

To close that issue we should also update documentation/CLI output.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "format": "prettier --write \"**/*.{js,md,yml,ts,json,yaml}\"",
-    "lint": "prettier --list-different \"**/*.{js,md,yml,ts,json,yaml}\"",
+    "lint": "prettier --check \"**/*.{js,md,yml,ts,json,yaml}\"",
```
Comment (cabljac) [inline package.json:8]:
```suggestion
    "lint": "prettier --list-different \"**/*.{js,md,yml,ts,json,yaml}\"",
```

**Comment by cabljac** [comment]
Would like to add tests for this before it is approved, [since we have that capability now.](https://github.com/firebase/extensions/pull/1307) i have a concern about the solution, but i might be wrong.

**Comment by cabljac** [comment]
Tests still need some tidying, and could add a couple more.

**Comment by ciriousjoker** [comment]
I just tried out 1473d9bde73a3b50313fe96c6e187b993a19f089 and it works, but only after I manually remove the "sourceCollectionPath" question and put the value (`users/{uid}/workout`) directly into the variable by editing `config.js`.

Mby I'm just stupid, but trying to provide the value when getting asked about it via inquirer always resulted in "The collection path does not match the regular expression provided".

**Comment by cabljac** [comment]
@ciriousjoker thanks, i've made a change to fix that in this PR.

**Comment by cabljac** [comment]
closing in favour of updated PR https://github.com/firebase/extensions/pull/1414

**Comment by cabljac** [comment]
closing in favour of https://github.com/firebase/extensions/pull/1414

**Pull request closed without merging**